### PR TITLE
Documentation Update: Campaigns

### PR DESCRIPTION
**Pull request by ignaciopenia**
This PR impacts on [docs.xlink.network](https://docs.xlink.network/). The changes can be previewed at the temporary URL [here](https://coinfabrik.gitbook.io/xlink-docs/getting-started/campaigns).

A new section was created within **Getting Started** guide called **Campaigns**. It offers instructions and examples for users interested in participating in XLink campaigns. Below are the main changes performed:

- Added a introduction explaining what campaigns are and how to participate.
- Added the steps for participating in campaigns, including how to navigate to the Campaigns tab in the XLink app, register for a campaign and complete required actions (e.g., staking, adding liquidity).

### CODE REVIEW

File: getting-started/campaigns.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
+---
+description: Learn how to participate in XLink campaigns and earn rewards.
```
Comment (tenuki) [inline getting-started/campaigns.md]: que te parece: 
 ->  Learn **what are and** how to participate in XLink campaigns and earn rewards.
?

File: getting-started/campaigns.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    visible: true
+  pagination:
+    visible: false
+---
+
+# Campaign Guide
+
+## What are XLink Campaigns
+
+Welcome to the **Campaigns**! XLink campaigns are special events where users can participate in various actions on supported blockchains to earn rewards. These actions typically involve interacting with the XLink bridge, such as staking tokens, adding liquidity, or bridging assets between chains. Campaigns offer great opportunities to earn **ALEX**, **LiSTX**, **XLink Points**, and other project-specific tokens as rewards.
```
Comment (tenuki) [inline getting-started/campaigns.md]: These actions typically involve interacting with **many different actors in the STACKS ecosystem, like** the XLink bridge, ~~such as staking tokens, adding liquidity, or bridging assets between chains.~~ **and use them to accomplish different challenges.**

File: getting-started/campaigns.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+Each campaign may have different requirements and actions to complete, but they all aim to incentivize user participation and enhance Bitcoin DeFi ecosystems. Below, we'll explore the rewards you can earn and some tips on how to maximize your participation.
```
Comment (tenuki) [inline getting-started/campaigns.md]: que tal: 
"Each campaign ~~may have~~ **involves** different requirements and actions ~~to complete, but they~~ **, and** all aim to incentivize user participation and strengthen user confidence in the Bitcoin DeFi ecosystem"
?

Comment (tenuki) [reply getting-started/campaigns.md]: "Below, we'll explore the rewards you can earn and some tips on how to maximize your participation."
-> maximize your participation or the earnings?

File: getting-started/campaigns.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+## How to Participate in XLink Campaigns
+
+Participating in campaigns is easy and rewarding. Although each campaign might be slightly different, the basic steps to participate are similar. Here’s how you can get started:
+
+### Step 1: **Register on the Platform**
+First, visit the **XLink App** and navigate to the **Campaigns** tab. This is where you'll find all the active campaigns. 
+
+Look for the **See Details** button next to the campaign you're interested in. Clicking it will provide more details on how to participate, including specific steps and links to register.
+
+![Campaign Registration](../.gitbook/assets/campaigns/campaigns-menu.png)
```
Comment (tenuki) [inline getting-started/campaigns.md]: Esta imagen mas que Registration parece Selection/Browsing/Navigation alguna de esas no?

File: getting-started/campaigns.md [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+
+Once you've selected a campaign, you'll need to register for it by following the instructions provided under the **How to Participate** section. You might be directed to specific platforms like **CORE**, **Mode**, or **Stacks**, depending on the campaign.
+
+For example, in the **ALEX Staking Campaign**, you'll be directed to stake your ALEX tokens on LISA. Follow the campaign-specific instructions to complete the staking process and earn rewards.
+
+![ALEX Staking Campaign Example](../.gitbook/assets/campaigns/alex-staking-example.png)
+
+For example, in the **ALEX Liquidity Campaign**, you’ll be directed to add liquidity on a supported DeFi platform. Be sure to follow the specific steps outlined in each campaign.
```
Comment (tenuki) [inline getting-started/campaigns.md]: No habria que usar de vuelta  For example..   podriamos cambiar por: 
"**Similarly,** in the **ALEX Liquidity Campaign**, you’ll be directed to add liquidity on a supported DeFi platform. Be sure to follow the specific steps outlined in each campaign."

**Review by tenuki — CHANGES_REQUESTED** [review]

File: getting-started/campaigns.md (hunk 3 above)
Comment (ignaciopenia) [reply getting-started/campaigns.md:22]: I meant to take advantage of the participation opportunities that exist. But it is also related to getting more rewards..

Comment (ignaciopenia) [reply getting-started/campaigns.md]: > "Below, we'll explore the rewards you can earn and some tips on how to maximize your participation." -> maximize your participation or the earnings?

I meant to take advantage of the participation opportunities that exist. But it is also related to getting more rewards..

Comment (tenuki) [reply getting-started/campaigns.md]: good!

**Review by tenuki — APPROVED** [review]
all good

**Pull request merged**